Architecture.md

```diff
@@ -71,7 +77,7 @@ It is used because it discovers when containers are started/stopped and can dyna
 ### Definitions
 
 - the Service refers to PaaSTech as a whole;
-- a Client is a user account created by an end user against the Service;
+- a Client (also referred to as a User) is a user account created by an end user against the Service;
 - a Project is a materialisation of a Git repository, created by a Client using either the web frontend or the CLI. A Project can be deployed by the Client by pushing its code to the Service.
```

Comment:
What does materialization mean in this context ?

Reply:
This is a requested thing, as User has been used a lot in place of Client.

Reply:
I meant the sentence "a Project is a materialisation of a Git repository"